CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Creating releases
+
+If you are a maintainer of the repository and would like to tag a new release, follow these steps:
+
+- Send a heads-up in the Slack channels or post an issue with plan to tag a new release
```

Comment:
Should we specify `#rules_ios` in the official Bazel Slack?

Reply:
```suggestion
- Send a heads-up in the Slack ([#rules_ios](https://bazelbuild.slack.com/archives/C04UA35PAMR)) channel or post an issue with plan to tag a new release
```

Reply:
Added!